### PR TITLE
Add AzureCLI 2.60.0

### PR DESCRIPTION
**Pull request by azclibot**
Add AzureCLI 2.60.0
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151599)

### CODE REVIEW

**Comment by stephengillie** [comment]
URL: https://azcliprod.azureedge.net/msi/azure-cli-2.60.0.msi 
No such host is known. (azcliprod.azureedge.net:443)

(Automated message - build 878)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> URI: https://azcliprod.azureedge.net/msi/azure-cli-2.60.0-x64.msi, Validation result: Failed, Error Message: No such host is known. (azcliprod.azureedge.net:443) No such host is known.
 

(Automated response - build 878.)

File: manifests/m/Microsoft/AzureCLI/2.60.0/Microsoft.AzureCLI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.AzureCLI
+PackageVersion: 2.60.0
+Installers:
+- Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://azcliprod.azureedge.net/msi/azure-cli-2.60.0.msi
```
Comment (PckgrTom) [inline manifests/m/Microsoft/AzureCLI/2.60.0/Microsoft.AzureCLI.installer.yaml:9]: URL has changed to [https://azcliprod.blob.core.windows.net/msi/azure-cli-2.60.0.msi](https://azcliprod.blob.core.windows.net/msi/azure-cli-2.60.0-x64.msi)

File: manifests/m/Microsoft/AzureCLI/2.60.0/Microsoft.AzureCLI.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  InstallerSha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+- Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://azcliprod.azureedge.net/msi/azure-cli-2.60.0-x64.msi
```
Comment (PckgrTom) [inline manifests/m/Microsoft/AzureCLI/2.60.0/Microsoft.AzureCLI.installer.yaml:13]: URL has changed to https://azcliprod.blob.core.windows.net/msi/azure-cli-2.60.0-x64.msi

**Pull request closed without merging**